.github/workflows/canary-integration-test.yml

```diff
@@ -552,6 +557,7 @@ jobs:
       run: |
         # set VERSION to a dummy value since Jenkins normally sets it for us. Do this to make Helm happy and not fail with "Error: Invalid Semantic Version"
         GOPATH=$(go env GOPATH) make clean && make -j$nproc IMAGES='ceph' VERSION=0 build
+        tests/scripts/validate_modified_files.sh build
```

Comment:
With #7677 can this verification be added in the build helper so we won't need to call this script everywhere?

Reply:
@travisn yes